package.json

```diff
@@ -34,8 +34,9 @@
     "typescript": "^4.9.4"
   },
   "scripts": {
-    "compile-wasm": "turbo run compile",
-    "build": "pnpm run compile-wasm && turbo run build",
+    "pnpm:devPreinstall": "pnpm --filter @anonklub/merkle-tree compile && pnpm --filter @anonklub/spartan-ecdsa compile",
```

Comment:
let's improve with
```suggestion
    "pnpm:devPreinstall": "pnpm --filter @anonklub/merkle-tree --filter @anonklub/spartan-ecdsa --parallel compile",
```

Reply:
bad idea
after testing this breaks the CI action because of temporary file lock: we must not parallelize.
Somehow i didn;t have this issue locally so I that it was a good way to speed up the action